src/Service/RevisionSaver.php

```diff
@@ -36,6 +37,8 @@ public function __construct( MediawikiApi $api ) {
 	 * @returns bool success
 	 */
 	public function save( Revision $revision, EditInfo $editInfo = null ) {
+		$editInfo = $editInfo ? $editInfo : $revision->getEditInfo();
+		
```

Comment:
Now they are gone. My Atom usually removes them automatically, but not when my cursor is in the line when saving.